src/main/java/core/basesyntax/List.java

```diff
@@ -1,7 +1,7 @@
 package core.basesyntax;
 
 public interface List<T> {
-    void add(T value);
+    boolean add(T value);
```

Comment:
Do not change this interface. Task is **to implement** interface - not change it.